ansible/roles/backup2l/templates/etc/cron.daily/zz-backup2l.j2

```diff
@@ -0,0 +1,15 @@
+{# Copyright (C) 2021 DebOps <https://debops.org/>
+ # SPDX-License-Identifier: GPL-3.0-only
+ #}
+#!/bin/bash
+
+# {{ ansible_managed }}
+
+# The following command invokes 'backup2l' with the default configuration
+# file (/etc/backup2l.conf).
+#
+# (Re)move it or this entire script if you do not want automatic backups.
+#
+# Redirect its output if you do not want automatic e-mails after each backup.
+
+! command -v backup2l > /dev/null || nice -n 19 backup2l -b{% if backup2l__no_cron_emails %} >/dev/null{% endif %}
```

Comment:
This command will not pass through `shellcheck`. You should define a variable with `backup2l__no_cron_emails` as a value, and then on the shell level interpret that value.

I think that compressing logic like this in the shell script is counter-intuitive. The scripts are meant to be understandable by humans that audit the code. I would go with a traditional `if` / `else` block here.